channel-adapter/channel-adapter.yml

```diff
@@ -0,0 +1,5 @@
+tagline: Object to interact with channels
+desc: The channel adapter is an interface to interact with a channel.
+helps: Ballerina supports a rich set of Connectors to interact with different messaging channels.
```

Comment:
s/messaging channels/applications/
also mention users can create their own adapters and make them available for others.